cmd/neofs-node/reputation.go

```diff
@@ -356,10 +404,33 @@ func (s *loggingReputationServer) SendLocalTrust(_ context.Context, req *v2reput
 	return resp, nil
 }
 
-func (s *loggingReputationServer) SendIntermediateResult(_ context.Context, req *v2reputation.SendIntermediateResultRequest) (*v2reputation.SendIntermediateResultResponse, error) {
+func (s *reputationServer) SendIntermediateResult(_ context.Context, req *v2reputation.SendIntermediateResultRequest) (*v2reputation.SendIntermediateResultResponse, error) {
 	resp := new(v2reputation.SendIntermediateResultResponse)
 
 	// todo: implement me
 
 	return resp, nil
 }
+
+func (s *reputationServer) Address() string {
```

Comment:
Why do we need this and `PublicKey()` methods?

Reply:
i decided to pass `reputationServer` to `passedRoute` as is, not internal `reputationServer.cfg`, because i thought that `cfg` could be gone sometime in the future => hided localserver info behind method `Address()` but not field `cfg`

but maybe it was the wrong decision

Reply:
I agree with your motivation, for such a need, in order to avoid an additional method at the moment, you can embed `cfg` into `reputationServer`.

Reply:
ok, added